test/support/integration/enterprise_server.ex

```diff
@@ -0,0 +1,186 @@
+defmodule LivebookTest.Integration.EnterpriseServer do
```

Comment:
```suggestion
defmodule LivebookTest.EnterpriseServer do
```

could be under `EnterpriseIntegration`, but it's probably too much.

Reply:
It's a good suggestion. @josevalim wdyt?